.github/workflows/cdk-test.yml

```diff
@@ -0,0 +1,36 @@
+name: cdk test
```

Comment:
Generally I prefer GHA but I note that the rest of this project is using TeamCity for CI. Using GHA therefore has a couple of drawbacks in this context:

1. It means developers need to think about 2 CI platforms when working on a single repo
2. It means that you can deploy a change with Riff-Raff _even when this test is failing_

You could probably make this a [required status check](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging) as a workaround for 2.

Reply:
👍 added cdk-test as a required status check